README.md

```diff
@@ -50,13 +53,14 @@ This can be done with the `--out` flag like in the following example.
 
 ## Flags
 
-| **flag**                    | **Description**                                                                    |
-|-----------------------------|------------------------------------------------------------------------------------|
-| -i, --in <dir>              | The src directory for strapi                                                       |
-| -o, --out <dir>             | The output directory to output the types to                                        |
-| --component-prefix <prefix> | A prefix for components                                                            |
-| -D, --delete-old            | CAUTION: This option is equivalent to running rm -rf on the output directory first |
-| --prettier <file>           | The prettier config file to use for formatting typescript interfaces               |
+| **flag**                    | **Description**                                                                      | default   |
```

Comment:
Can you put the default column in bold as well?

Reply:
Sure. I also made the headers all capitalized for consistency.